Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```diff
@@ -240,8 +240,21 @@
   parent: Protolathe
   id: SecurityTechFab
   name: Security TechFab
-  description: Prints security equipment and some basics.
+  description: Prints equipment for use by security crew.
   components:
+  - type: Sprite
+    netsync: false
+    sprite: Structures/Machines/security_techfab.rsi
+    layers:
+    - state: icon
+      map: ["enum.ProtolatheVisualLayers.Base"]
+    - state: unlit
+      shader: unshaded
+      map: ["enum.ProtolatheVisualLayers.BaseUnlit"]
+    - state: icon
+      map: ["enum.ProtolatheVisualLayers.AnimationLayer"]
+    - state: panel
+      map: ["enum.WiresVisualLayers.MaintenancePanel"]
```

Comment:
There's only 3 states on the sprite but 4 layers? I imagine you might want to remove the icon above the unlit later.

Reply:
That's because the lathe visualizer breaks without it